src/cat.ts

```diff
@@ -119,31 +112,32 @@ export class Cat {
   }
 
   private static validatePrior(priorDist: string, priorPar: number[], minTheta: number, maxTheta: number) {
-    if (priorDist !== 'norm' && priorDist !== 'unif') {
-      throw new Error('The prior distribution you provided is not supported');
-    }
     if (priorDist === 'norm') {
       if (priorPar.length !== 2) {
-        throw new Error('The prior distribution parameters you provided are not valid');
+        throw new Error(`The prior distribution parameters should be an array of two numbers. Received ${priorPar}.`);
       }
-      if (priorPar[1] <= 0) {
-        throw new Error('The prior distribution standard deviation you provided is not valid');
+      const [mean, sd] = priorPar;
+      if (sd <= 0) {
+        throw new Error(`Expected a positive prior distribution standard deviation. Received ${sd}`);
       }
-      if (priorPar[0] < minTheta || priorPar[0] > maxTheta) {
-        throw new Error('The prior distribution mean you provided is not valid');
+      if (mean < minTheta || mean > maxTheta) {
+        throw new Error(`Expected the prior distribution mean to be between the min and max theta. Received mean: ${mean}, min: ${minTheta}, max: ${maxTheta}`);
       }
-    }
-    if (priorDist === 'unif') {
+      return normal(mean, sd, minTheta, maxTheta);
+    } else if (priorDist === 'unif') {
       if (priorPar.length !== 2) {
-        throw new Error('The prior distribution parameters you provided are not valid');
+        throw new Error(`The prior distribution parameters should be an array of two numbers. Received ${priorPar}.`);
       }
-      if (priorPar[0] >= priorPar[1]) {
-        throw new Error('The uniform distribution bounds you provided are not valid (min must be less than max)');
+      const [minSupport, maxSupport] = priorPar;
+      if (minSupport >= maxSupport) {
+        throw new Error(`The uniform distribution bounds you provided are not valid (min must be less than max). Received min: ${minSupport} and max: ${maxSupport}`);
       }
-      if (priorPar[0] < minTheta || priorPar[1] > maxTheta) {
-        throw new Error('The uniform distribution bounds you provided are not within theta bounds');
+      if (minSupport < minTheta || maxSupport > maxTheta) {
+        throw new Error(`The uniform distribution bounds you provided are not within theta bounds. Received minTheta: ${minTheta}, minSupport: ${minSupport}, maxSupport: ${maxSupport}, maxTheta: ${maxTheta}.`);
       }
+      return uniform(minSupport, maxSupport, 0.1, minTheta, maxTheta);
     }
+    return [];
```

Comment:
Thanks Anya. These changes are great!. I just have one question about this line. As you've written it now, the `priorDist` parameter to `Cat` accepts any value. Like if I pass in `priorDist = "binomial"`, I won't get an error, I'll just get an empty array for the prior. I think it would be better if we threw an error if `priorDist` wasn't `"norm"` or `"unif"`. That way, the user knows early on that they did something wrong and won't be expecting some other behavior. I like how you had it before where it threw an error like

```suggestion
    throw new Error(`priorDist must be "unif" or "norm." Received ${priorDist} instead.`);
```

Also (and this is a very small nitpick), do you think we could accept either "norm" or "normal" and "unif" or "uniform?"